ci/release_python.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+set -e -u -o pipefail
+
+if [[ -n "${DEBUG-}" ]]; then
+  set -x
+fi
+
+cd "$(dirname "$0")/"
+
+PYPI_TEST_SECRET="keanu-eng-test-pypi"
+PYPI_REAL_SECRET="keanu-eng-pypi"
```

Comment:
These variable names suggest that they are secrets in plain text - I assume you mean they are the names of the secrets, that are stored in the vault?

Reply:
Yeah good point. You're correct they are the names of the secrets, I have changed the naming to reflect this.